distribution/openrct2.d.ts

```diff
@@ -2873,6 +2874,11 @@ declare global {
          */
         numSeats: number;
 
+        /**
+         * Whether this car seats guests in pairs (as opposed to single riders).
+         */
+        usedInPairs: boolean;
```

Comment:
Maybe it's clearer to call it `seatedInPairs` or `usesSeatsInPairs`, in the context of a vehicle?

Reply:
That'd warrant renaming the method `Vehicle::IsUsedInPairs` as well, which I'd be fine with, but I was just following the precedent set by that method's name.

Reply:
It's okay if the api naming doesn't match internal, IMHO it's more important that it's clear to plugin creators who may not know the internal structure or naming schemes.

Reply:
It's not hard to refactor the internal names since there's only a few instances where this function is used. I'll go ahead and switch them over as well. I think it's important to maintain consistency, even if it's something this small.